tools/CreateRuleFabricBot/CreateRuleFabricBot/Rules/PullRequestLabel/PullRequestLabelFolderCapability.cs

```diff
@@ -132,8 +139,41 @@ private List<CodeOwnerEntry> ReadOwnersFromFile()
             {
                 while ((line = sr.ReadLine()) != null)
                 {
-                    CodeOwnerEntry entry = CodeOwnerEntry.Parse(line);
-                    if (entry != null)
+                    // Does the line start with '# PRLabel: "label1", "label2"
+
+                    // Remove tabs and trim extra whitespace
+                    line = line.Replace('\t', ' ');
+                    line = line.Trim();
+
+                    // Empty line, move on
+                    if (string.IsNullOrEmpty(line))
+                    {
+                        continue;
+                    }
+
+                    CodeOwnerEntry entry = new CodeOwnerEntry();
+
+                    // if we have the moniker in the line, parse the labels
+                    if (line.IndexOf(CodeOwnerEntry.LabelMoniker, System.StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
Not sure it matters much but this does mean it will try to fine the labels on a line that is not a comment. While it might work here that is technically an broken CODEOWNERS file.

Reply:
Not sure what you mean. Do you have an example?

Reply:
This would match a line like:
```
PRLabel: %label
```

Basically you aren't validating that the label is on its own line that is a comment.

Reply:
Gotcha. Resolved